tests/e2e-pw/specs/merchant/merchant-orders-manual-capture.spec.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { getShopper, getMerchant } from '../../utils/helpers';
+import {
+	goToOrder,
+	goToWooPaymentsSettings,
+} from '../../utils/merchant-navigation';
+import { saveWooPaymentsSettings } from '../../utils/merchant';
+import {
+	emptyCart,
+	fillCardDetails,
+	focusPlaceOrderButton,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import { goToShop } from '../../utils/shopper-navigation';
+
+/**
+ * Local variables.
+ */
+let orderId;
+
+test.describe( 'Order > Manual Capture', () => {
+	test.beforeEach( async ( { browser } ) => {
+		// Merchant go to settings, enable capture later, and then save.
+		const { merchantPage } = await getMerchant( browser );
+		await goToWooPaymentsSettings( merchantPage );
+		await merchantPage.getByTestId( 'capture-later-checkbox' ).click();
```

Comment:
This can lead to flaky tests because it just clicks the checkbox instead of unchecking it, which does the opposite of what we intend if it's already checked.

We are already following a good pattern to do it properly in [the merchant utils](https://github.com/Automattic/woocommerce-payments/blob/00599cd3c8365d27e33ebd8a823568c0caadee61/tests/e2e-pw/utils/merchant.ts#L319).

Reply:
I tried updating this with functions in the `merchant.ts` file, but ran into some issues. The changes were in https://github.com/Automattic/woocommerce-payments/pull/10180/commits/5b28c7bd0a1ba07110a61a667fa554c39cab209d and I brought up my issues in slack: p1737475317185709-slack-C01B8KNUYSW

I ended up having to change `check` to `click` in order to get the tests to work. Maybe you will be able to see something I didn't? https://github.com/Automattic/woocommerce-payments/pull/10180/commits/71fc6dfdb1e158aea37d246d69ff45b4e8bf1d5b

Reply:
Yes, this is expected in this case because we're preventing the default behaviour of the checkbox from displaying the confirmation modal, but Playwright expects the checkbox to be checked after clicking it. In this case, our only solution is to use `click` to activate it.

Reply:
Okay, that makes sense. Comment has been removed.